data/data-home.json

```diff
@@ -150,6 +144,12 @@
             "linkedin":"https://www.linkedin.com/in/hanutsingh",
             "picurl":"images/alumini/default.jpeg"
           },
+          {
+            "name":"SHIVAM RAJPUT",

```

Comment:
Change from UPPERCASE to Sentence Case (if that's the word for it)

Reply:
Was just following the format everything else was also capital.